libs/au-pair-cost/feature/src/lib/au-pair-cost.component.scss

```diff
@@ -0,0 +1,355 @@
+ion-card {
+  background-color: transparent;
+  box-shadow: none;
+}
+
+ion-card-header{
+  padding-bottom: 0 !important;
+}
+
+.summary-div-wrapper {
+  text-align: center;
+  width: 100%;
+  position: relative;
+  display: block;
+}
+
+.summary-div {
+  width: 100%;
+  height: 94px;
+  display: grid;
+  border-radius: 16px;
+  grid-template-columns: repeat(4, 1fr);
+  align-items: center;
+  justify-items: center;
+  margin: auto;
+  font-size: 24px;
+  margin-bottom: 60px;
+  background-color: var(--ion-color-tertiary);
+}
+
+.au-pair-name {
+  background-color: var(--ion-color-primary);
+  color: white;
+  margin: 0;
+  height: 100%;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  font-size: 24px;
+}
+
+.hourly-rate {
+  color: white;
+  margin: 0;
+  height: 100%;
+  display: flex;
+  border-left: 1px var(--ion-background-color) solid;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  font-size: 24px;
+}
+
+.total-hours {
+  color: white;
+  margin: 0;
+  height: 100%;
+  display: flex;
+  border-left: 1px var(--ion-background-color) solid;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  font-size: 24px;
+}
+
+.total-remuneration {
+  color: white;
+  margin: 0;
+  height: 100%;
+  display: flex;
+  border-left: 1px var(--ion-background-color) solid;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  border-style: solid;
+  border-right: none;
+  border-top: none;
+  border-bottom: none;
+}
+
+.ion-padding {
+    padding: 40px;
+    width: 100%;
+    position: relative;
+    display: inline-block;
+}
+
+.graphs-grid{
+  display: grid;
+  gap: 80px;
+  grid-template-columns: minmax(800px,0.6fr) minmax(500px,0.4fr);
+}
+
+.card-bar-graph {
+  border-radius: 16px;
+  height: 517px;
+  background-color: white;
+  font-size: var(--heading-size);
+  margin: 0;
+}
+
+.card-pie-graph {
```

Comment:
This class is the exact same as above. Please give it a general name and assing one class to both